### PR TITLE
[IBCDPE-1017] Adds Vesteinn Thorsson (ISB) to `iatlas-project`

### PR DESCRIPTION
**Pull request by BWMac**
This PR adds access to the `iatlas-project` workspace for Vesteinn Thorsson (PI at ISB) and removes access for Clarisse Lau (also ISB).

### CODE REVIEW

File: templates/nextflow-tower-config-bucket.yaml [hunk 1]
```diff
@@ -69,6 +69,7 @@ Resources:
                 - "*@sagebionetworks.org"
                 - "cheimann@isbscience.org"
                 - "clarisse.lau@isbscience.org"
```
Comment (thomasyu888) [inline templates/nextflow-tower-config-bucket.yaml]: Can we remove Clarisse as well? Not sure if it should be a separate PR.

**Review by thomasyu888 — APPROVED** [review]
🔥 Pre-approving! Thank you!

File: templates/nextflow-tower-config-bucket.yaml (hunk 1 above)
Comment (BWMac) [reply templates/nextflow-tower-config-bucket.yaml]: I think it's fine if we do it in one
